packages/cli/__tests__/compile-stylex-folder-test.js

```diff
@@ -123,16 +123,21 @@ describe('cli works with multiple inputs and outputs', () => {
     watch: false,
     babelPresets: [],
   };
+  afterAll(() => clearTestDir(config));
   test('script compiles multiple directories', (done) => {
     const onClose = () => {
-      let snapshotDir = './snapshot';
+      let isSecondOutput = false;
       for (const dir of config.output) {
         if (dir.endsWith('src2')) {
-          snapshotDir = './snapshot2';
+          isSecondOutput = true;
```

Comment:
Have you hardcoded support for up to 2 folders?

Reply:
This is only the mocks used for a simple test, it supports more than 2 directories.